### ISSUE TITLE
Finalize Wireframes for Home page

### ISSUE THREAD

**Issue by kristinashu** [mozillascience/site#95]
Based on wireframes created in https://github.com/mozillascience/site/issues/87 and seen here https://redpen.io/p/kwe0b1b8415da9f6f4

We will need to update the wireframes once we get current site analytics, user IA feedback and copy.

**Comment by cassiemc** [comment]
@kristinashu Can you link directly to the most up-to-date wireframe here? Not sure what I should be looking at in the big folder - is it this one? https://redpen.io/xc4c6fca695501df65

**Comment by kristinashu** [comment]
@cassiemc here is the most recent version https://redpen.io/xc4c6fca695501df65

**Comment by hannahkane** [comment]
Blocked on final content and IA

**Comment by my-le** [comment]
Last iteration for Home Page on Red Pen: https://redpen.io/bg9672bcf28b51ac0c

![msl-home-wireframe-lyt01-p06](https://cloud.githubusercontent.com/assets/10426601/13536325/876c3258-e23f-11e5-8a32-888464764f84.jpg)

**Comment by hannahkane** [comment]
Final content is [here](https://docs.google.com/document/d/1k2Ds-iw4B_Sh5ae1areW1qfuLPtSqmgohtXYBkOi7r4/edit#heading=h.sr1d8andeat3).

Final IA also selected.

No longer blocked!

**Comment by kristinashu** [comment]
Here are two slightly different versions of the home page wireframes https://redpen.io/p/vee57da2ed47dfc6a4

**Comment by kristinashu** [comment]
I've made a few updates to the home page wireframes: https://redpen.io/p/vee57da2ed47dfc6a4

Option 3 (https://redpen.io/yzb9bb323c70144dab) is based on the initial feedback on Options 1 and 2.

cc @hannahkane @kaythaney @acabunoc @auremoser @stephwright @zee-moz @cassiemc

**Comment by kristinashu** [comment]
I think we're getting very close to finalizing this https://redpen.io/xh59fdf36751be7ae1

Just a few minor outstanding items:
- can everyone review the intro copy and please add suggestions? It needs to be concise but still properly reflect both what MSL is and what open science is
- there was a comment a little while ago on whether or not we're going to continue to use the [forum](https://forum.mozillascience.org/), any thoughts? 
- we're still not sure if events should be a main level menu item or under programs but I will explore both directions in a separate ticket

![science-wireframes-05_home 2x](https://cloud.githubusercontent.com/assets/1068783/13684597/548533ae-e6c1-11e5-9c4b-d3cc5207c91a.jpg)

**Comment by kaythaney** [comment]
thanks!

re: discourse - we have a number of resources there (dusty ones, but there
nevertheless). @acabunoc - if we do away with discourse (which i'm not
against), where would those be migrated to?

On Thu, Mar 10, 2016 at 1:14 PM, Kristina <notifications@github.com> wrote:

> I think we're getting very close to finalizing this
> https://redpen.io/xh59fdf36751be7ae1
>
> Just a few minor outstanding items:
>
>    - can everyone review the intro copy and please add suggestions? It
>    needs to be concise but still properly reflect both what MSL is and what
>    open science is
>    - there was a comment a little while ago on whether or not we're going
>    to continue to use the forum <https://forum.mozillascience.org/>, any
>    thoughts?
>    - we're still not sure if events should be a main level menu item or
>    under programs but I will explore both directions in a separate ticket
>
> [image: science-wireframes-05_home 2x]
> <https://cloud.githubusercontent.com/assets/1068783/13684597/548533ae-e6c1-11e5-9c4b-d3cc5207c91a.jpg>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozillascience/site/issues/95#issuecomment-195052023>.
>



-- 
@kaythaney <http://twitter.com/kaythaney/>
@MozillaScience <http://twitter.com/mozillascience>

**Comment by acabunoc** [comment]
I hadn't thought about migration.

We could try to migrate them to github issues, but I'm not convinced it's the best plan.

**Comment by kaythaney** [comment]
blog posts as resources? or separate resources, where appropriate? at the
very least, we should assess what should be archived if we discontinue.

On Fri, Mar 11, 2016 at 8:39 AM, Abigail Cabunoc Mayes <
notifications@github.com> wrote:

> I hadn't thought about migration.
>
> We could try to migrate them to github issues, but I'm not convinced it's
> the best plan.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozillascience/site/issues/95#issuecomment-195443434>.
>



-- 
@kaythaney <http://twitter.com/kaythaney/>
@MozillaScience <http://twitter.com/mozillascience>

**Comment by kristinashu** [comment]
Closing this ticket. If we want to remove the forum, we can open a new ticket to sort out migration of the content.
